.circleci/config.yml

```diff
@@ -91,7 +122,9 @@ jobs:
             kubectl config use-context staging
       - deploy:
           name: Deploy fala to staging
-          command: .circleci/deploy_to_kubernetes staging
+          command: |
+            .circleci/deploy_to_kubernetes staging $ECR_DOCKER_REGISTRY/$ECR_DOCKER_IMAGE-nginx nginx
+            .circleci/deploy_to_kubernetes staging $ECR_DOCKER_REGISTRY/$ECR_DOCKER_IMAGE-webapp webapp
```

Comment:
🙋‍♂️ `$ECR_DOCKER_IMAGE` has 3 values throughout the file, which might get confusing to follow. Should we create a single one that is used throughout? Everywhere else we could use the postfix variant, eg. `${ECR_DOCKER_IMAGE}-nginx`.

We no longer seem to have a use for `$ECR_DOCKER_REGISTRY` as a separate environment as it's always in conjunction with `$ECR_DOCKER_IMAGE`. Should we merge them into a single one?

Reply:
Sorry, I shouldn't have committed these changes. The pull request is only meant to contain changes around build and pushing docker images to ECR repos.

I'll do Kubernetes changes in next PR.